Scripts/parameters/demand/hb_business_long.json

```diff
@@ -132,47 +132,51 @@
             "attraction": {},
             "impedance": {},
             "log": {
-                "logsum": 0.7201
+                "logsum": 0.61849
             },
             "individual_dummy": {}
         },
         "train": {
-            "constant": -0.93174,
+            "constant": -1.19903,
             "generation": {},
             "attraction": {},
             "impedance": {},
             "log": {
-                "logsum": 0.7201
+                "logsum": 0.61849
             },
             "individual_dummy": {}
         },
         "long_d_bus": {
-            "constant": -1.24571,
+            "constant": -1.63808,
             "generation": {},
             "attraction": {},
             "impedance": {},
             "log": {
-                "logsum": 0.7201
+                "logsum": 0.61849
             },
             "individual_dummy": {}
         },
         "car_work": {
-            "constant": 1.32378,
+            "constant": -0.14918,
             "generation": {},
-            "attraction": {},
+            "attraction": {
+                "car_density": 3.62549
+            },
```

Comment:
This should be on in the generation group for clarity. But it should not affect results in this kind of model structure.